docs/standard/serialization/how-to-determine-if-netstandard-object-is-serializable.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "How to: Determine if a .NET Standard object is serializable"
+ms.custom: ""
+ms.date: "10/20/2017"
+ms.prod: ".net"
+ms.topic: "article"
+dev_langs:
+- "csharp"
+- "vb"
+helpviewer_keywords: 
+  - "serializing objects"
+  - "objects, serializing steps"
+author: "rpetrusha"
+ms.author: "ronpet"
+manager: "wpickett"
+---
+# How to: Determine if a .NET Standard object is serializable
+
+The .NET Standard is a specification that defines the types and members that must be present on specific .NET implementations that conform to that version of the standard. However, the .NET Standard does not define whether a type is serializable. The types defined in the .NET Standard Library are not marked with the <xref:System.SerializableAttribute> attribute. Instead, specific .NET implementations, such as the .NET Framework and .NET Core, are free to determine whether a particular type is serializable. 
+
+If you've developed a library that targets the .NET Standard, your library can be consumed by any .NET implementation that supports the .NET Standard. This means that you cannot know in advance whether a particular type is serializable; you can only determine whether it is serializable at run time.
+
+You can determine whether an object is serializable at runtime by retrieving the value of the <xref:System.Type.IsSerializable> property of a <xref:System.Type> object that represents that object's type. The following example provides one implementation. It defines an `IsSerializable(Object)` extension method that indicates whether any <xref:System.Object> instance can be serialized.
+
+[!code-csharp[is-a-type-serializable](~/samples/snippets/standard/serialization/is-serializable/csharp/program.cs#2)]
```

Comment:
this path doesn't look right for snippets unless we want to change it. it should go under the language folders first and then you give a name
so under https://github.com/dotnet/docs/tree/master/samples/snippets/csharp and under https://github.com/dotnet/docs/tree/master/samples/snippets/visualbasic